docs/source/general/finetuning.rst

```diff
@@ -0,0 +1,90 @@
+**********
+Finetuning
+**********
+
+Finetuning (or transfer-learning) is the process of tweaking a model trained on a large dataset, to your particular (likely much smaller) dataset. All Flash tasks have a pre-trained backbone that was trained on large datasets such as ImageNet, and that allows to decrease training time significantly.
+
+.. note:: If you have a huge dataset and prefer to train from scratch.
+
+When using :func:`~flash.Trainer.finetune`, you also need to provide a finetune `strategy`.
+
+.. code-block:: python
+
+    import flash
+    from flash import download_data
+    from flash.vision import ImageClassificationData, ImageClassifier
+
+    # 1. download and organize the data
+    download_data("https://download.pytorch.org/tutorial/hymenoptera_data.zip", 'data/')
+
+    data = ImageClassificationData.from_folders(
+        train_folder="data/hymenoptera_data/train/",
+        valid_folder="data/hymenoptera_data/val/"
+    )
+
+    # 2. build the task, and turn off pre-training
+    task = ImageClassifier(num_classes=2, pretrained=False)
+
+    # 3. train!
+    trainer = flash.Trainer()
+    trainer.finetune(model, data, strategy="no_freeze")
+
+
+Finetune options
+================
+
+Flash finetune `strategy` argument can either a string or an instance of :class:`~python_lightning.callbacks.finetuning.BaseFinetuning`.
+Furthermore, Flash supports 4 builts-in Finetuning Callback accessible via those strings:
+
+* `no_freeze`: Don't freeze anything.
+* `freeze`: Freeze the backbone parameters when training starts.
+* `freeze_unfreeze`: Freeze the backbone parameters when training starts and unfreeze the backbone when reaching `unfreeze_epoch`.
+* `unfreeze_milestones`: Freeze the backbone parameters when training starts and unfreeze the end backbone when reaching first milestones and begining when reaching second one.
```

Comment:
can you rephrase? Also, do we have any reference to this technique?

Reply:
Done ! Not really, but I have seen it several time.